engine/src/main/java/com/datatorrent/stram/StreamingContainerAgent.java

```diff
@@ -215,11 +219,11 @@ public void addOperatorRequest(StramToNodeRequest r)
           for (PTOperator.PTInput input : out.sinks) {
             // Create mappings for all non-inline operators
             if (input.target.getContainer() != out.source.getContainer()) {
-              InputPortMeta inputPortMeta = getIdentifyingInputPortMeta(input);
-              StreamCodec<?> streamCodecInfo = getStreamCodec(inputPortMeta);
-              Integer id = physicalPlan.getStreamCodecIdentifier(streamCodecInfo);
+              final StreamCodec<?> streamCodec = getIdentifyingInputPortMeta(input).getStreamCodec();
+              final Integer id = physicalPlan.getStreamCodecIdentifier(streamCodec);
```

Comment:
Why the addition of finals in these two lines, does it specify anything significant?